sp-installer-menu/menu/menu.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://json-schema.org/draft-07/schema",
   "$id": "https://schemas.conda.io/menuinst-1.schema.json",
-  "menu_name": "Scientific Python (#PKG_VERSION#)",
+  "menu_name": "#FOLDER_AND_CATEGORY_NAME#",
```

Comment:
use a variable to set both `menu_name` and `Categories`. Until https://github.com/conda/menuinst/issues/343 is fixed, it's necessary that they be the same.